specification/trafficmanager/resource-manager/Microsoft.Network/stable/2018-04-01/examples/HeatMapState-GET.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "azuresdkfornetautoresttrafficmanager1323",
+    "profileName": "azuresdkfornetautoresttrafficmanager3880",
+    "heatMapType": "state",
+    "api-version": "2018-04-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/{subscription-id}/resourceGroups/azuresdkfornetautoresttrafficmanager1323/providers/Microsoft.Network/trafficManagerProfiles/azuresdkfornetautoresttrafficmanager3880/heatMaps/stateTrafficFraction",
```

Comment:
name value coming from "id" does not match the value in "name" property. Please make sure service returns the correct "id" and "name" value

Reply:
updated name and type - do we need a separate API version?